_posts/2022-08-19-kubeflow-user-survey-2022.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "Kubeflow User Survey 2022"
+layout: post
+toc: false
+comments: true
+image: images/logo.png
+hide: false
+categories: [release]
+permalink: /kubeflow-user-survey-2022/
+author: "Kubeflow 1.6 Release Team"
+---
+
+In May 2022, the Kubeflow user survey was opened to gather community feedback. The goal of the survey was to understand the adoption of Kubeflow and to gather input on the benefits, gaps, and requirements for machine learning use cases.
+
+The survey was comprised of 24 questions (multiple choice and freeform). It ran from May 6th to June 7th and received 151 responses. Out of the 151 responses, 91 responses provided optional feedback on how to improve Kubeflow and the Community. In the 2022 survey, we received an increased number of freeform answers, which provided additional insights into users’ needs.
+
+## Key Takeaways
+
+- 85% of the user deploy more than one Kubeflow component
```

Comment:
I think user should be plural i.e. 85% of the users deploy more than one Kubeflow component

Reply:
Done